internal/pkg/manifest/validate.go

```diff
@@ -151,6 +152,18 @@ func (l LoadBalancedWebService) validate() error {
 	}); err != nil {
 		return fmt.Errorf("validate unique exposed ports: %w", err)
 	}
+	exposedPortsIndex, err := l.ExposedPorts()
+	if err != nil {
+		return err
+	}
+	if err = validateHealthCheckPorts(validateHealthCheckPortsOpts{
+		exposedPorts:      exposedPortsIndex,
+		mainContainerPort: l.ImageConfig.Port,
+		alb:               &l.HTTPOrBool.HTTP,
+		nlb:               &l.NLBConfig,
+	}); err != nil {
+		return fmt.Errorf("validate load balancer health check ports: %w", err)
```

Comment:
Does it make more sense to be
```suggestion
	if err = validateALBHealthCheckPorts(validateHealthCheckPortsOpts{
		exposedPorts:      exposedPortsIndex,
		mainContainerPort: l.ImageConfig.Port,
		alb:               &l.HTTPOrBool.HTTP,
	}); err != nil {
		return fmt.Errorf("validate application load balancer health check ports: %w", err)
```
And do the same thing to NLB separately? Because `validateHealthCheckPorts` seems to deal ALB and NLB separately.